packages/cli/fragments/fragment-vanilla-ts/src/vite-env.d.ts

```diff
@@ -1,7 +1,7 @@
-import type * as TauriApiTypes from "@tauri-apps/api";
+export {};
```

Comment:
why not just remove the whole line?

Reply:
Without it you get this error:
> Augmentations for the global scope can only be directly nested in external modules or ambient module declarations.

And `export {}` was the only thing i found on the internet to get rid of it. [1](https://stackoverflow.com/a/59499895) / [2](https://bobbyhadz.com/blog/typescript-augmentations-for-global-scope-can-only-be-directly-nested)

Reply:
right, just remove `declare global` and `export {};`. this should be enough
```ta
interface Window {
  greet(): void;
}
```